### PR TITLE
e339 refactor: update initialize with compose plugin and update structure

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: source/includes/connect/plugins/login-button/example/_import-script-compose.md.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+> If need to start multiple plugins, also import Endpass connect Compose plugin as html script tag,
```
Comment (lamartire) [inline source/includes/connect/plugins/login-button/example/_import-script-compose.md.erb]: `import` is not correct wording for that case. I guess:
```
If you need to start multiple plugins just add following scripts in your html markup,
```

File: source/includes/connect/plugins/oauth/_options.md.erb [hunk 2]
```diff
@@ -6,6 +6,6 @@ The following options can be passed when initializing the OAuth plugin.
 
 |Name|Type|Description|
 |----|----|-----------|
-|**oauthClientId**|string|The client ID of your application.
+|**clientId**|string|The client ID of your application.
```
Comment (lamartire) [inline source/includes/connect/plugins/oauth/_options.md.erb]: I think, we should use `prettier` for markdown here. Very difficult to manage now

File: source/includes/connect/_quick-start-compose.md.erb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+```javascript
+// For example, when you need multiple plugins usage, just combine them by EndpassComposePlugin
+
+// If you are using modules or npm
+import EndpassComposePlugin from '@endpass/connect/compose';
+import EndpassOauthPlugin from '@endpass/connect/oauth';
+
+// Insert your client ID here
+const YOUR_CLIENT_ID = '';
```
[the remark this answers is not in the document]
Comment (kazurus) [reply source/includes/connect/_quick-start-compose.md.erb]: it's requirement

File: source/includes/connect/_quick-start-compose.md.erb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+```javascript
+// For example, when you need multiple plugins usage, just combine them by EndpassComposePlugin
+
+// If you are using modules or npm
```
[the remark this answers is not in the document]
Comment (kazurus) [reply source/includes/connect/_quick-start-compose.md.erb]: @lamartire why?

**Comment by lamartire** [comment]
@ifiri ready to approve

File: source/includes/connect/plugins/oauth/_options.md.erb (hunk 2 above)
Comment (dudiq) [reply source/includes/connect/plugins/oauth/_options.md.erb]: ah, we have `ruby`, not `node` with `package.json`, soo it's not so easy to implement.
maybe in next time.

File: .gitattributes [hunk 5]
```diff
@@ -0,0 +1,119 @@
+
```
Comment (kazurus) [inline .gitattributes]: please, add vscode to gitignore and delete

File: source/includes/connect/example/_import-browser-oauth.md.erb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+> Import Endpass connect OAuth plugin as html script tag
```
Comment (kazurus) [inline source/includes/connect/example/_import-browser-oauth.md.erb]: we have separate folder for oauth (connect/plugins/oauth/expample). Can we move to it?

File: source/includes/connect/_errors.md.erb [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ## Errors
 
-<%= partial "includes/connect/example/\_connect-errors.md.erb" %>
```
[the remark this answers is not in the document]
Comment (dudiq) [reply source/includes/connect/_errors.md.erb]: hmm, it's working for me

[the remark this answers is not in the document]
Comment (kazurus) [reply source/includes/connect/_errors.md.erb]: If it's working without slash it's better without

**Pull request merged**